src/main/scala/com/monsanto/arch/cloudformation/model/resource/ElasticLoadBalancingV2.scala

```diff
@@ -0,0 +1,408 @@
+  * @param DependsOn Declare dependencies for resources that must be created or deleted in a specific order.
+  */
+case class `AWS::ElasticLoadBalancingV2::TargetGroup`(
+  name:                       String,
+  Protocol:                   ALBProtocol,
+  Port:                       Token[Int],
+  VpcId:                      Token[ResourceRef[`AWS::EC2::VPC`]],
+  Matcher:                    Option[Matcher],
+  HealthCheckIntervalSeconds: Option[Token[Int]] = None,
+  HealthCheckPath:            Option[Token[String]] = None,
+  HealthCheckPort:            Option[Token[String]] = None,
+  HealthCheckProtocol:        Option[ALBProtocol] = None,
+  HealthCheckTimeoutSeconds:  Option[Token[Int]] = None,
+  HealthyThresholdCount:      Option[Token[Int]] = None,
+  UnhealthyThresholdCount:    Option[Token[Int]] = None,
+  Name:                       Option[Token[String]] = None,
+  TargetGroupAttributes:      Option[Seq[TargetGroupAttribute]] = None,
+  Targets:                    Option[Seq[TargetDescription]] = None,
+  Tags:                       Option[Seq[AmazonTag]] = None,
+  override val Condition:     Option[ConditionRef] = None,
+  override val DependsOn:     Option[Seq[String]] = None
+) extends Resource[`AWS::ElasticLoadBalancingV2::TargetGroup`] with HasArn {
+  def when(newCondition: Option[ConditionRef] = Condition): `AWS::ElasticLoadBalancingV2::TargetGroup` = copy(Condition = newCondition)
+  def arn: Token[String] = ResourceRef(this)
+
+  def loadBalancerArns: Token[String] = `Fn::GetAtt`(Seq(name, "LoadBalancerArns"))
+  def targetGroupFullName: Token[String] = `Fn::GetAtt`(Seq(name, "TargetGroupFullName"))
+}
+
+object `AWS::ElasticLoadBalancingV2::TargetGroup` extends DefaultJsonProtocol {
+  implicit val format: JsonFormat[`AWS::ElasticLoadBalancingV2::TargetGroup`] = jsonFormat18(`AWS::ElasticLoadBalancingV2::TargetGroup`.apply)
+}
+
+
+sealed trait ALBProtocol
+object ALBProtocol extends DefaultJsonProtocol {
+  case object HTTP  extends ALBProtocol
+  case object HTTPS extends ALBProtocol
+  val values = Seq(HTTPS, HTTPS)
+  implicit val format: JsonFormat[ALBProtocol] = new EnumFormat[ALBProtocol](values)
+}
+
+
+/**
+  * @see http://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html
+  */
+sealed trait ELBSecurityPolicy
+object ELBSecurityPolicy extends DefaultJsonProtocol {
+  case object `ELBSecurityPolicy-2016-08`         extends ELBSecurityPolicy
+  case object `ELBSecurityPolicy-TLS-1-2-2017-01` extends ELBSecurityPolicy
+  case object `ELBSecurityPolicy-TLS-1-1-2017-01` extends ELBSecurityPolicy
+  case object `ELBSecurityPolicy-2015-05`         extends ELBSecurityPolicy
+  val values = Seq(`ELBSecurityPolicy-2016-08`, `ELBSecurityPolicy-TLS-1-2-2017-01`, `ELBSecurityPolicy-TLS-1-1-2017-01`, `ELBSecurityPolicy-2015-05`)
+  implicit val format: JsonFormat[ELBSecurityPolicy] = new EnumFormat[ELBSecurityPolicy](values)
+}
+
```

Comment:
Looks like I should be able to add support for arbitrary values by overriding the `stringifier` in `EnumFormat`. I'll give that a whirl.